lib/live.tsx

```diff
@@ -0,0 +1,47 @@
+async function getCurrentTasks(req) {
+  const urlRoute = req
+    ? /* Serverside */ process.env.URL_BASE + "api/points/tasks"
+    : /* Client */ "/api/points/tasks";
+  console.log(urlRoute);
```

Comment:
are we commiting console logs? or no care

Reply:
Imma clean it up, just got editing working